docs/concepts/what-is-an-activity.md

```diff
@@ -26,10 +26,15 @@ In its simplest form, a Temporal Activity is a function or an object method in o
 Temporal does not recover Activity state in case of failures.
 Therefore an Activity function is allowed to contain any code without restrictions.
 
-Activities are invoked asynchronously through task queues.
-A task queue is essentially a queue used to store an Activity task until it is picked up by an available worker.
-The worker processes an Activity by invoking its implementation function.
+#### Invoke Activities through Task Queues
+
+Activities are invoked asynchronously through Task Queues.
+A Task Queue is essentially a queue used to store an Activity task until it is picked up by an available Worker.
+The Worker processes an Activity by invoking its implementation function.
 When the function returns, the worker reports the result back to the Temporal service which in turn notifies the Workflow about completion.
+
+#### Implement an Activity asynchronously
+
 It is possible to implement an Activity fully asynchronously by completing it from a different process.
 
 - An Activity can be implemented as a synchronous method or fully asynchronously involving multiple processes.
```

Comment:
This sentence isn't necessary. The user coming here most likely knows about sync vs async